.gitignore

```diff
@@ -7,6 +7,7 @@
 /data/sqlite/*.db
 /public/all.*
 /public/selfoss-sw-offline.js
+/public/package-lock.json
```

Comment:
Are you refering to .gitignore?

Reply:
> We should start committing this file.

Are you refering to .gitignore?

Reply:
> Are you refering to .gitignore?

Yes. Since selfoss is an application, we should pin the dependencies uing the lock file. I will commit

Reply:
> Are you refering to .gitignore?

Yes. Since selfoss is an application, we should pin the dependencies uing the lock file. I will commit it in https://github.com/SSilence/selfoss/pull/1096, which deals with changing the dependencies.

Reply:
> > Are you refering to .gitignore?
> 
> Yes. Since selfoss is an application, we should pin the dependencies uing the lock file. I will commit it in #1096, which deals with changing the dependencies.

So, what should I do as of now?

Reply:
I would just revert this commit for now.

Reply:
> I would just revert this commit for now.

Ok, no problem.